.github/workflows/publish-release-artifact.yml

```diff
@@ -7,9 +7,14 @@ on:
 permissions:
   contents: read # to fetch code (actions/checkout)
 
+concurrency: 
+  group: ${{github.workflow}} - ${{github.ref}}
+  cancel-in-progress: true  # Cancel release if release is started again for same tag. 
+
```

Comment:
Uploading a release seems like a bad choice to interrupt, lest a partial upload is mistakenly left due to the interruption.
```suggestion

```

Reply:
Thank you for the feedback! Out of curiosity, currently if you push to the same version causing this pipeline to run again, does it override the old release with the new one and throw the old one away?